### PR TITLE
MGMT-2277 - remove references to the unused ignition-generate-image

### PR DESCRIPTION
**Pull request by ronniel1**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -3199,8 +3199,6 @@ definitions:
       pull_secret_set:
         type: boolean
         description: True if the pull secret has been added to the cluster.
-      ignition_generator_version:
```
Comment (filanov) [inline swagger.yaml:3202]: Can you please open a ticket about deleting it from the DB? 
Nick added a nice way to do it with https://github.com/openshift/assisted-service/tree/master/internal/migrations

Comment (ronniel1) [reply swagger.yaml:3202]: added https://issues.redhat.com/browse/MGMT-3095

**Review by filanov — APPROVED** [review]

**Comment by YuviGold** [comment]
@ronniel1 You need to rebase

**Review by YuviGold — APPROVED** [review]

**Pull request merged**